master/buildbot/worker/ec2.py

```diff
@@ -523,6 +542,7 @@ def _request_spot_instance(self):
             raise LatentWorkerFailedToSubstantiate()
```

Comment:
these lines are marked uncovered by unit tests. Please add unit tests for it, so that we can keep the code working.